engines/scumm/sound.cpp

```diff
@@ -584,10 +586,23 @@ void Sound::startTalkSound(uint32 offset, uint32 b, int mode, Audio::SoundHandle
 			size = result->compressed_size;
 #endif
 		} else {
-			offset += 8;
+			// WORKAROUND
+			// The speech sample at offset 0x76ccbca ("Hey you!") which is used when
+			// Indy gets caught on the German submarine seems to not be a VOC but plain
+			// PCM s16be at 44.1 kHz with a bogus VOC header.
+			// To work around this we play this sample using the raw PCM decoder.
+			// Fixes Trac#10559
+			if (mode == 2 && (_vm->_game.id == GID_INDY4) && (_vm->_language == Common::EN_ANY) && offset == 0x76ccbca) {
```

Comment:
What about size check also, as a precaution?

Reply:
Which size?

Reply:
size of the in-game sample, the one which you're overriding with 86016.

Reply:
I don't override it because it's not provided in the first place, the script only references an offset when playing a sample. `size` is otherwise only used when dealing with a compressed `monster.sou` where `size` comes from the hard-coded offset index table. I have to set it because the `SeekableSubReadStream` c-tor requires a size. `makeVOCStream` (normal code path) on the other hand doesn't because the VOC decoder has to parse the entire VOC to find the size of the audio data.

Reply:
(in other words, there is no size to check)